css/templates/lune/components/_header.css

```diff
@@ -0,0 +1,68 @@
+@import "../page-vars";
+
+.ampstart-headerbar {
+  justify-content: space-between;
+
+  height: var(--layout-header-height-mobile);
+
+  /* Override ampstart default box shadow styles */
+  box-shadow: none;
+
+  /* header is fixed so should be moved onto a new GPU layer */
+  will-change: transform;
+}
+
+.ampstart-headerbar-home-link {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.ampstart-headerbar-title {
+  font-size: var(--h4);
+  font-weight: var(--font-weight-black);
+
+  letter-spacing: var(--letter-spacing-4);
+  line-height: var(--line-height-1);
+
+  /* Using same indent as the letter spacing to make the text properly centered. */
+  text-indent: var(--letter-spacing-4);
+}
+
+.ampstart-navbar-trigger {
+  padding-left: var(--space-1);
+
+  font-size: var(--h3);
+}
+
+@media (--breakpoint-mt) {
+  .ampstart-navbar-trigger {
+    padding-left: var(--space-3);
+  }
+
+  .ampstart-headerbar {
+    height: var(--layout-header-height-desktop);
+  }
+}
+
+/* Remove the default margin top from amp-sidebar and navbar */
+.ampstart-headerbar+:not(amp-sidebar),
+.ampstart-headerbar+amp-sidebar+* {
+  margin-top: 0;
+}
+
+/* Target links on in the right fixed container of the navbar,
+   These should not be visible on mobile */
+.ampstart-headerbar-fixed a {
```

Comment:
where ever you feel like you need changes, you can always pass in class groups via JSON

like {{classes}} - general purpose classes
{{layout-classes}}  - ablsolute, relative, float etc
{{spacing-classes}} - p0, m0, mx-auto etc
{{resposive-display-classes}} - sm-hide md-hide
{{resposive-layout-classes}} -  md-col12, sm-col6  etc

If there are classes already you could make them defaults and pass in your classes via JSON